check_limits.py

```diff
@@ -1,18 +1,11 @@
-
-def battery_is_ok(temperature, soc, charge_rate):
-  if temperature < 0 or temperature > 45:
-    print('Temperature is out of range!')
-    return False
-  elif soc < 20 or soc > 80:
-    print('State of Charge is out of range!')
-    return False
-  elif charge_rate > 0.8:
-    print('Charge rate is out of range!')
-    return False
-
-  return True
-
-
+from raise_alert_for_battery_parameters import *
+
+limit = {
```

Comment:
`limit` is not used?

Reply:
yes limit is being used. Depending on the limit value only the messages are defined.

![image](https://user-images.githubusercontent.com/13776900/110654429-20ef1c80-81e4-11eb-885e-6385337ef5a9.png)